# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../iso-codes/pkgconfig-path.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- iso-codes-0.50/Makefile.am.orig	2006-03-12 21:50:47.000000000 +0100
+++ iso-codes-0.50/Makefile.am	2006-03-12 21:51:09.000000000 +0100
@@ -1,6 +1,6 @@
 SUBDIRS = iso_639 iso_3166 iso_4217 notes
 
-pkgconfigdir = $(datadir)/pkgconfig
+pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = iso-codes.pc
 
 EXTRA_DIST = \
